### DISCUSSION TITLE
问一个关于推理速度的问题

### DISCUSSION THREAD

**Discussion by lingyezhixing** [zai-org/ChatGLM3#267] [General]
假如我要使用int4量化，那么直接加载int4模型与加载全量模型再量化两者的推理速度在理论上是否会有差别？为什么我在实际使用中感觉前者的推理速度极快而后者的推理速度和全量模型没什么明显区别

**Comment by zRzRzRzRzRzRzR** [comment]
1. 能提供一下你的说的int4量化的方法吗
2. ChatGLM3-6B 没有对应的int4官方模型，您是怎么对比的，我们复现一下

**Comment by zRzRzRzRzRzRzR** [comment]
> 假如我要使用int4量化，那么直接加载int4模型与加载全量模型再量化两者的推理速度在理论上是否会有差别？为什么我在实际使用中感觉前者的推理速度极快而后者的推理速度和全量模型没什么明显区别

给一个邮箱，讨论一下
